callisto/delivery/models.py

```diff
@@ -12,36 +10,40 @@
 from django.db import models
 from django.template import Context, Template
 from django.utils import timezone
-from django.utils.crypto import get_random_string, pbkdf2
+from django.utils.crypto import get_random_string
 from django.utils.html import strip_tags
 
+from callisto.delivery.hashers import get_hasher, identify_hasher
+
+# change this to the value previously used
+ORIGINAL_KEY_ITERATIONS = 100
```

Comment:
actually now I'm thinking we should create an ORIGINAL_KEY_ITERATIONS settings value to be set in existing instances and then subclassed hasher.must_update can pull from that in the prefix empty string case